packages/web-core/src/services/SessionService.ts

```diff
@@ -125,13 +125,7 @@ export class SessionService {
     const sessionParts = this.#shortSession.value.split('.');
     const sessionPayload = JSON.parse(atob(sessionParts[1]));
 
-    return {
-      email: sessionPayload.email,
-      name: sessionPayload.name,
-      orig: sessionPayload.orig,
-      sub: sessionPayload.sub,
-      exp: sessionPayload.exp,
-    };
+    return sessionPayload;
```

Comment:
this change is because the payload from BE of session user has changed (after adding new identifiers) and it is better for the using function to be able to use entire object (as I needed info from it initially when I was using it for login state).